.circleci/config.yml

```diff
@@ -19,7 +19,33 @@ jobs:
       - image: circleci/node:10.15
     steps:
       - checkout
-      - run: echo test
+      - run:
+          name: Run yarn
+          command: |
+            yarn
+            yarn build:api
+            yarn build:frontend
+      - run: 
+          name: Run eslint - frontend
```

Comment:
We might want the test commands as scripts in the root directly instead of doing this changing in the config.yml like you did with yarn build:api and yarn build:frontend. Something like yarn test:api and yarn test:frontend